README.md

```diff
@@ -34,6 +34,18 @@ can be found [here](https://docs.google.com/document/d/163L7pv6w5Z38rUrl2EwRJq-A
 from coursescheduler import generate_schedule
 schedule = generate_schedule(professors, schedule)
 ```
+A third parameter, `debug`
+
+There are also functions that will validate input data according to the spec:
```

Comment:
I think we should validate, but there's an argument to be made that it's the caller's responsibility. I told Russ we'd validate the inputs and he said he would as well, before calling the algorithm. I don't think it needs to be done twice, but it's not the end of the world as long as there isn't much overhead. I made these functions "public" and documented them because I thought they might be useful for back-end's tests, not because I thought back-end would validate inputs before calling the algorithm(s).

Reply:
Relevant resource: https://softwareengineering.stackexchange.com/questions/187715/validation-of-the-input-parameter-in-caller-code-duplication

Reply:
Do we want to add the code for validation on this PR? The other place it would make sense is in the error handling PR.